tasks/README.md

```diff
@@ -6,4 +6,8 @@ This directory contains the tasks that are part of this benchmark.
 Name | Summary | Category
 ---- | ----------- | --------
 `task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
\ No newline at end of file
+`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
+`task481_BANKING77_classification` | Check the category of the banking query | Classification
```

Comment:
Task 481: `Givan a sentence, classify the sentence based on the needed banking operation.`
Task 482: I'm not sure about this task. Please update the summary like the other examples after modifying the definition.
Task 483: `Givan a set of descriptions, write how many different ways have been used to express the same thing.`
Task 484: `Given a text, write a keyword about the text.`